app/src/main/java/org/mozilla/fenix/components/toolbar/BrowserToolbarController.kt

```diff
@@ -206,12 +207,28 @@ class DefaultBrowserToolbarController(
                         .show()
                 }
             }
-            ToolbarMenu.Item.AddToHomeScreen, ToolbarMenu.Item.InstallToHomeScreen -> {
+            ToolbarMenu.Item.AddToHomeScreen -> {
```

Comment:
`isInstallable()` corresponds to what actually happens, while these toolbar items just repesent the option shown on screen. Let's keep the when statement as it and move the bookmark shortcut call to the `else` branch.